source/_integrations/squeezebox.markdown

```diff
@@ -100,13 +100,45 @@ Call a custom Squeezebox JSON-RPC API. The result of the query will be stored in
+| `parameters`   | yes      | Array of additional parameters to pass to Lyrion Music Server (p1, ..., pN in the CLI documentation). |
 
 This action can be used to integrate a Squeezebox query into an automation. For example, in a Python script, you can get a list of albums available by an artist like this:
 `hass.services.call("squeezebox", "call_query", { "entity_id": "media_player.kitchen", "command": "albums", "parameters": ["0", "20", "search:beatles", "tags:al"] })`
 To work with the results:
 `result = hass.states.get("media_player.kitchen").attributes['query_result']`
+
+### Action `search`
+
+This action allows you to search the LMS music library for albums, artists, genres, tracks and playlists. You can also search for favorites and players. The result of the search will be stored in the 'query_result' attribute of the player.
+
+| Data attribute  | Optional | Description                                                                                                                       |
+| --------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `entity_id`     | no       | Name(s) of the Squeezebox entities where to run the search.                                                                       |
+| `command`       | no       | Items to search on the Lyrion Music Server. This must be one of albums, artists, genres, tracks, playlists, favorites or players. |
+| `return_items`  | no       | The maximum numbers of items to return.                                                                                           |
+| `search_string` | yes      | Limit the search to those items matching the search string                                                                        |
+| `tags`          | yes      | Specify tags you wish the search to return, overriding the default tag list                                                       |
+
+This action can be used to integrate a Squeezebox query into an automation. For example, in a Python script, you can get a list of the first 20 albums containing the word "classic" like this:
+`hass.services.call("squeezebox", "search", { "entity_id": "media_player.kitchen", "command": "albums", "return_items": "20", "search_string":"classic"] })`
+To work with the results:
+`result = hass.states.get("media_player.kitchen").attributes['query_result']`
+
+### Action `play`
+
+This action allows you to play music from the LMS music library. You can play an album, artist, genre, track, playlist or favorite. It has two forms. You can either search for an item using full text searching for the name of the item, or you can specify the item_id of the item, which is most likely retrieved previously using the search action. If you use the text search, your search must return a single result, which will be played or added to the current playlist.
```

Comment:
```suggestion
This action allows you to play music from the LMS music library. You can play an album, artist, genre, track, playlist or favorite. It has two forms. You can either search for an item by name, or you can specify the item_id, which is most likely retrieved previously using the search action. If you search by name, your search must return a single result, which will be played or added to the current playlist.
```
I don't understand "your search must return a single result". How do I know what the search returns, before I even search? And how can I ensure there will be only one match, if I don't have the unique ID?